### DISCUSSION TITLE
Please help me code hidden header scroll down in shopify

### DISCUSSION THREAD

**Discussion by dimasfaiz** [community/community#62741] [Programming Help]
### Select Topic Area

Question

### Body

Hello my name is dimas faiz, i want to ask for this case header in Shopify. I want, my header hidden when scroll down and also hidden scroll down, any ideas code for this code in my Shopify? Please let me know, thank you
![Screen Shot 2023-08-05 at 17 19 13](https://github.com/community/community/assets/24537417/b2a719f0-6b18-442e-8c80-7fd7a4d474ff)

**Comment by hossein-rg** [comment]
Hey buddy, here's the optimal approach for dealing with JavaScript:

```javascript
window.addEventListener('scroll',function(){
let header = document.querySelector(".header-wrapper");
let scrollPos = window.scrollY;
if(scrollPos > 100 ) {
header.classList.add("header-hidden");
}else{
header.classList.remove("header-hidden");
}
});
``` 
```css
.header-hidden{
transform:translateY(-100%)
}
```

**Reply by dimasfaiz** [reply]
Sure, Can I copy this code for what is the section? theme.liquid? or base.css?

Thank you
